website/docs/basics/blocks.md

```diff
@@ -3,3 +3,50 @@ id: blocks
 title: Blocks
 sidebar_label: Blocks
 ---
+
+Since WordPress 5.0, we were introduced to the new editing experience using block. With blocks, the idea is to have the same look and feel in the editor as it is on the frontend. As a developer, you just got a new technology that you should know if you want to do a great project on WordPress. That new technology is React because everything regarding blocks is done using React. 
+
+Don't get discouraged just yet, because the WordPress team has made the process of creating blocks so easy. But we are not here to talk about the simple stuff. We want to do more complex projects with custom blocks, so follow along and see what we have prepared here.
+
+> Everything regarding the blocks you can read in the [WordPress documentation](https://developer.wordpress.org/block-editor/tutorials/block-tutorial/). We will not be explaining how everything natively works in this documentation, but instead, we will describe only how to use our setup and how it correlates with the native one. So everything that you can't find here, please check the WordPress Handbook.
+
+## Type of blocks
+
+WordPress Block Editor team has provided us with two different kinds of blocks:
```

Comment:
```suggestion
WordPress Core Editor team has provided us with two different kinds of blocks:
```

The team is actually called core editor since it is in the core now.